### PR TITLE
Another Print Edit

### PR DESCRIPTION
**Pull request by necrozis**
Hide all(?) huge jsons.
Recommended some prints.

### CODE REVIEW

File: working.py [hunk 1]
```diff
@@ -124,10 +124,11 @@ def encount_and_catch_pokemon(pokemon,api,position,config):
 	time.sleep(5)
 def _transfer_low_cp_pokemon(api,value,pokemon):
 	if 'cp' in pokemon and pokemon['cp'] < value:
-		print('need release this pokemon({}): {}'.format(value,pokemon))
+		#print('need release this pokemon({}): {}'.format(value,pokemon))
+		print('# Pokemon relased.')
```
Comment (eggins) [inline working.py]: Please fix up the spelling here

Comment (necrozis) [inline working.py]: Sorry for my English :dash:

**Comment by eggins** [comment]
Im going to wait until I can work on this again.

I will also be re-writing the styling for it - so each little log will look different! :)

**Comment by necrozis** [comment]
Ok :smile:

**Pull request closed without merging**